distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table_Conversions.enso

```diff
@@ -107,7 +126,7 @@ Table.from_objects value fields=Nothing =
 
             Table.new (used_fields.zip used_values)
         _ : Array -> Table.from_objects (Vector.from_polyglot_array value) fields
-        _ -> Error.throw (Illegal_Argument.Error "Invalid value for Table.from_objects. Currently must be one of JS_Object, Vector, Array, Number, Boolean, Text and Nothing are supported (got "+(Meta.get_simple_type_name value)+").")
+        _ -> Error.throw (Illegal_Argument.Error "Invalid value for Table.from_objects. Currently must be one of JS_Object, Vector, Array, Number, Boolean, Text, Date, Time and Nothing are supported (got "+(Meta.get_simple_type_name value)+").")
```

Comment:
```suggestion
        _ -> Error.throw (Illegal_Argument.Error "Invalid value for Table.from_objects. Currently must be one of JS_Object, Vector, Array, Number, Boolean, Text, Date, Date_Time, Time_Of_Day and Nothing are supported (got "+(Meta.get_simple_type_name value)+").")
```

There is no type `Time` in Enso (well, apart from `Value_Type.Time`, but here we are seemingly listing Enso types - so IMO its misleading to list a type that does not really exist).

If we want a shorter form, maybe?
```suggestion
        _ -> Error.throw (Illegal_Argument.Error "Invalid value for Table.from_objects. Currently must be one of JS_Object, Vector, Array, Number, Boolean, Text, date/time and Nothing are supported (got "+(Meta.get_simple_type_name value)+").")
```

This way we use a 'category name' for this group of types.